obsolete: license tier (#89)
avo-hq/avodocs: docs/3.0/licensing.md

## Add the license key

> After you purchase an Avo license, add it to your `config/initializers/avo.rb` file on the `license_key`.

Reviewer: ```suggestion
After you purchase an Avo license, add it to your `config/initializers/avo.rb` file under `license_key`.
```
